README.md

```diff
@@ -45,9 +45,16 @@ end of the file.
 
 <if there are any>
 
+## DNS Operator/Vendor Response
```

Comment:
`## DNS Violator Response` maybe? :) since it can be an implementation/software/system also.